onnx/backend/test/case/node/__init__.py

```diff
@@ -15,8 +15,74 @@
 from ..utils import import_recursive
 from ..test_case import TestCase
 
+
 _NodeTestCases = []
 
+from onnx.onnx_pb import NodeProto, AttributeProto
+from onnx.onnx_operators_pb import FunctionProto
+
+
+def function_expand_helper(node,  # type: NodeProto
+                           function_proto,  # type: FunctionProto
+                           op_prefix  # type:  Text
+                           ):  # type:  (...) -> List[NodeProto]
+    node_list = []
+    input_names_map = dict()
+    output_names_map = dict()
+    attribute_map = node.attribute
+
+    for idx in range(len(node.input)):
+        if idx not in range(len(function_proto.input)):
+            continue
```

Comment:
Isn't this an error-situation? May be it would be better to return an empty list, instead of continuing, if there is an error like this. Similarly, what if for a function_proto.input there is no matching node.input? In that case, I think we should treat it as an empty string (a missing optional input).

Reply:
tbh the checker in gtests should catch this later. so however we handle the mismatched # of io tensors is fine here